docs/site/todo-tutorial-putting-it-together.md

```diff
@@ -112,6 +112,13 @@ including SOAP or REST services. Continue to the bonus section to learn how
 LoopBack connectors make it super easy to fetch data from other services and
 [enhance your Todo application with location-based reminders](todo-tutorial-geocoding-service.md).
 
+### The next step: TodoList Application
+
+If you would like to try out using some of the more advanced features of
```

Comment:
We also have a bonus step below ... this part seems a bit disjointed. How about making a section called `Next Steps` and then saying to try out the to learn about adding an external service for geolocation to the Todo App. Or try out the `todolist` tutorial to learn about relations in LoopBack 4.